tests/Frontend/views/rooms/Index.spec.js

```diff
@@ -256,7 +256,7 @@ describe('Room Index', function () {
         message: 'Test'
       }
     });
-    moxios.stubRequest('/api/v1/roomTypes', {
+    moxios.stubRequest('/api/v1/roomTypes?filter=searchable', {
```

Comment:
Is there a test to check if the searchable filter is set if the user doesn't have the permission and not if the user has the permission to view all rooms?